### ISSUE TITLE
System time change results in change of delta time 

### ISSUE THREAD

**Issue by QbieShay** [godotengine/godot#23566]
**Godot version:**
`4fca3fe1`

**OS/device including version:** 
Debian 9.2

**Issue description:**
The time tracking funcion yields inconsistant result if a system time change happens when godot is running.
The time is calculated as follows:
```
var total_time

func _ready():
    total_time = 0

func _process(delta):
    total_time =+ delta
    print(str(total_time))
```

**Steps to reproduce:**
Attach the previous script to a node and run the scene. Change the system clock when the scene is running and the time in the console will show very big negative numbers.

**Comment by akien-mga** [comment]
CC @Faless

**Comment by akien-mga** [comment]
>     total_time =+ delta

Typo in the example or in your test project? :)

**Comment by QbieShay** [comment]
Eheh sorry, I rewrote the piece of code out if a way bigger script

**Comment by QbieShay** [comment]
@Faless told me that it is solved through #22424 and #22564. Closing.

**Comment by akien-mga** [comment]
Ah right I didn't notice you were on an older checkout.
